src/app/page.tsx

```diff
@@ -249,49 +269,60 @@ export default function Home() {
-      <div className="w-full">
-        <button
-          className="bg-gray-300 hover:bg-gray-400 text-gray-700 font-semibold py-1 px-2 border-gray-400 rounded"
-          onClick={() => setShowSettings((prev) => !prev)}
-        >
-          Settings
-        </button>
-      </div>
-
-      {showSettings ? (
-        <SettingsModal
-          settings={settings}
-          handleChange={handleSettingsChange}
-          toggleModal={() => setShowSettings((prev) => !prev)}
-        />
-      ) : (
-        <></>
-      )}
-
-      <h1>HDRICalibrationTool</h1>
+    <main className="bg-white flex min-h-screen flex-col items-center justify-between">
       <div>
-        <h2>Image Upload</h2>
+        <nav className="pt-10 bg-gray-300 fixed left-0 w-1/4 h-full">
+          <ul>
+            <li className="font-bold pt-5 pl-5">Navigation Configuration</li>
+            <li className="pt-5 pl-5">
+              <a href="#image_selection">Image Selection</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#response">Response File</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#c_r_v">Cropping, Resizing, and View Settings</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#v">Vignetting Correction</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#nd">Neutral Density Correction</a>
+            </li>
+            <li className="pt-5 pl-5">
+              <a href="#cf">Calibration Factor Correction</a>
+            </li>
+            <li className="pt-10 pl-5">
+              <button
+                className="bg-gray-700 hover:bg-gray-400 text-gray-300 font-semibold py-1 px-2 border-gray-400 rounded"
+                onClick={() => setShowSettings(!showSettings)}
+                >
+                Settings
+              </button>
+              {showSettings && (
+                <Settings
+                  settings={settings}
+                  handleChange={handleSettingsChange}
+                  toggleDialog={() => setShowSettings(!showSettings)}
+                />
+                )}
+            </li>
+            <li className="pt-5 pl-5">
+              <button
+                onClick={handleGenerateHDRImage}
+                className="bg-gray-700 hover:bg-gray-400 text-gray-300 font-semibold py-1 px-2 border-gray-400 rounded">
+                Generate HDR Image
+              </button>
+            </li>
 
+          </ul>
+        </nav>
+        <h1 className="font-bold pt-10">Configuration</h1>
+        <h2 className="font-bold pt-5" id="image_selection">Image Selection</h2>
         <button
           onClick={dialog}
-          className="bg-gray-300 hover:bg-gray-400 text-gray-700 font-semibold py-1 px-2 border-gray-400 rounded"
-        >
+          className="bg-gray-300 hover:bg-gray-400 text-gray-700 font-semibold py-1 px-2 border-gray-400 rounded">
```

Comment:
The image previews are still causing the app to scroll to the right where you can't see the left part of the app. I think if the image previews are wrapped, it shouldn't be an issue. Or the max width of the container for them is the width of the screen or something